src/overlays/actors/ovl_En_Dnt_Nomal/z_en_dnt_nomal.c

```diff
@@ -167,10 +167,9 @@ void EnDntNomal_Init(Actor* thisx, PlayState* play) {
         this->requiredObjectSlot = Object_GetSlot(&play->objectCtx, this->objectId);
         if (this->requiredObjectSlot < 0) {
             Actor_Kill(&this->actor);
-            // "What?"
-            PRINTF(VT_FGCOL(MAGENTA) " なにみの？ %d\n" VT_RST "\n", this->requiredObjectSlot);
-            // "Bank is funny"
-            PRINTF(VT_FGCOL(CYAN) " バンクおかしいしぞ！%d\n" VT_RST "\n", this->actor.params);
+            PRINTF(VT_FGCOL(MAGENTA) T(" なにみの？ %d\n", " What? %d\n") VT_RST "\n", this->requiredObjectSlot);
+            PRINTF(VT_FGCOL(CYAN) T(" バンクおかしいしぞ！%d\n", " The bank is weird! %d\n") VT_RST "\n",
```

Comment:
(for the first one)
The first word "nani" means what, "mino" is something else. So its "what ___ ?" But I cannot find anything online that says what it might translate to.

If we cant figure it out can just leave like this for now

Reply:
"I don't think "mino" means anything in particular
Or it could be a corrupted version of some other word (for example, "nani miteruno = what's watching?").
At the very least, this isn't meaningful Japanese on its own."

I think its fine to just leave it as "what" for now, I guess